megamek/src/megamek/server/Server.java

```diff
@@ -37490,20 +36129,22 @@ private void registerWithServerBrowser(boolean register, String urlString) {
                     }
                 }
             } else {
+                // TODO : Windchild Can this be removed?
                 while ((line = rd.readLine()) != null) {
                     // System.out.println(line);
                 }
             }
             rd.close();
             printout.close();
-        } catch (Exception e) {
+        } catch (Exception ignored) {
         }
     }
 
     public Set<Coords> getHexUpdateSet() {
         return hexUpdateSet;
     }
 
+    // TODO : Windchild can this be removed?
```

Comment:
This is never called, can it be removed?

Reply:
Looking at how it's used, I don't think there should be a setter for this.